src/olympia/migrations/924-more-accurate-count-of-approved-webextensions.sql

```diff
@@ -0,0 +1,13 @@
+TRUNCATE TABLE `addons_addonapprovalscounter`;
```

Comment:
This could be avoided by changing migration 923 instead of creating a new one, which hasn't been processed on stage/prod yet. Thoughts ?